polaris/benchmark/__init__.py

```diff
@@ -1,11 +1,9 @@
-from polaris.benchmark._base import BenchmarkSpecification
-from polaris.benchmark._definitions import (
-    MultiTaskBenchmarkSpecification,
-    SingleTaskBenchmarkSpecification,
-)
+from polaris.benchmark._base import BenchmarkSpecification, BenchmarkV1Specification
+from polaris.benchmark._definitions import MultiTaskBenchmarkSpecification, SingleTaskBenchmarkSpecification
 
 __all__ = [
     "BenchmarkSpecification",
+    "BenchmarkV1Specification",
```

Comment:
Good question. I'd leave it as it, because V1 benchmarks were exposed through `SingleTaskBenchmarkSpecification` and `MultiTaskBenchmarkSpecification`.